src/domains/ioweb-common/06_cdn_itn.tf

```diff
@@ -47,3 +52,159 @@ module "io_web_profile_itn_fe_st" {
 
   tags = var.tags
 }
+
+resource "azurerm_cdn_frontdoor_profile" "portal_profile" {
+  name                = format("%s-profile-portal-afd-01", local.project_itn)
+  resource_group_name = azurerm_resource_group.io_web_profile_itn_fe_rg.name
+  sku_name            = "Standard_AzureFrontDoor"
+
+  tags = var.tags
+}
+
+data "azurerm_key_vault_certificate" "portal_custom_certificate" {
+  name         = "account-ioapp-it"
+  key_vault_id = module.key_vault.id
+}
+
+resource "azurerm_cdn_frontdoor_secret" "portal_certificate" {
+  name                     = "certificate"
+  cdn_frontdoor_profile_id = azurerm_cdn_frontdoor_profile.portal_profile.id
+
+  secret {
+    customer_certificate {
+      key_vault_certificate_id = data.azurerm_key_vault_certificate.portal_custom_certificate.id
+    }
+  }
+}
+
+resource "azurerm_cdn_frontdoor_custom_domain" "portal_custom_domain" {
+  name                     = format("%s-profile-fdd-01", local.project_itn)
+  cdn_frontdoor_profile_id = azurerm_cdn_frontdoor_profile.portal_profile.id
+
+  dns_zone_id = data.azurerm_dns_zone.ioapp_it.id
+  host_name   = "account.ioapp.it"
+
+  tls {
+    # Certificate managed by us and put in a kv
+    certificate_type        = "CustomerCertificate"
+    minimum_tls_version     = "TLS12"
+    cdn_frontdoor_secret_id = azurerm_cdn_frontdoor_secret.portal_certificate.id
+  }
+}
+
+resource "azurerm_cdn_endpoint" "portal_cdn_endpoint" {
+  name                = format("%s-profile-fde-01", local.project_itn)
+  profile_name        = azurerm_cdn_frontdoor_profile.portal_profile.name
+  location            = local.itn_location
+  resource_group_name = azurerm_resource_group.io_web_profile_itn_fe_rg.name
+
+  is_https_allowed = true
+  # a redirect to https is present on cdn_endpoint global rule
+  is_http_allowed = true
+
+  querystring_caching_behaviour = "BypassCaching"
+
+  origin {
+    name      = "primary"
+    host_name = module.io_web_profile_itn_fe_st.primary_web_host
```

Comment:
I'm not sure, but in `io-wallet` we used `primary_blob_host`

Reply:
`primary_blob_host` output is not currently available in the `dx` module. I followed the same variable used for the already established CDN.

Reply:
Ok, so it should work then. Probably the real value is actually the same. Please let me know if a change on DX module is needed. It is actually straightforward

Reply:
> Probably the real value is actually the same

following yesterday tests, I think that the value can be different. in fact, when enabling static website feature on storage account, a new hostname is created within the storage account. instead, I would expect a primary_blob_host to be `<name>.blob.core.windows` style